masterlist.yaml

```
@@ -4191,6 +4191,43 @@ plugins:
         util: '[TES4Edit v4.0.3h (Hotfix 1)](https://www.nexusmods.com/oblivion/mods/11536)'
         udr: 464
 
+  - name: 'LightSpell(Blue|EvilRed|LiteBlue|Torch|White(Dimmer)?)\.esp'
```

The regex could be simplified a bit:

```suggestion
  - name: 'LightSpell(EvilRed|(Lite)?Blue|Torch|White(Dimmer)?)\.esp'
```